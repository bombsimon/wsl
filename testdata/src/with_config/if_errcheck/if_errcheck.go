package testpkg

import "errors"

func fn1() {
	err := errors.New("x")
	if err != nil { // want "unnecessary whitespace decreases readability"
		panic(err)
	}
}

func fn2() {
	one := 1
	err := errors.New("x")
	if err != nil { // want "unnecessary whitespace decreases readability"
		panic(err)
	}

	_ = 1
}
