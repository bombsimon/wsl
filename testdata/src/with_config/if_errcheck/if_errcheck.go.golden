
package testpkg

import "errors"

func fn1() {
	err := errors.New("x")
	if err != nil { // want "unnecessary whitespace decreases readability"
		panic(err)
	}
}

func fn2() {
	a := 1

	err := errors.New("x")
	if err != nil { // want "unnecessary whitespace decreases readability"
		panic(err)
	}

	_ = a
}

func fn3() {
	a := 1

	err := errors.New("x") // want "missing whitespace decreases readability"
	if err != nil {
		panic(err)
	}

	_ = a
}

func fn4() {
	msg := "not an error"
	err := &msg

	if err != nil {
		panic(err)
	}
}
